### ISSUE TITLE
Microsoft.MicrosoftStickyNotes is getting removed

### ISSUE THREAD

**Issue by smith581** [Sycnex/Windows10Debloater#383]
I have tried to resolve this by re-adding
Get-AppxPackage -allusers Microsoft.MicrosoftStickyNotes | Foreach { Add-AppxPackage -DisableDevelopmentMode -Register "$($_.InstallLocation)\AppXManifest.xml" }
at the end of script to reinstall but it doesn't. It shows that it is supposed to be whitelisted but it isnt being saved. Resulting in an sccm redeploy after users called in about it being missed.....

**Comment by Sycnex** [comment]
Do you have a copy of the whitelist contents?

**Comment by smith581** [comment]
Do you have a copy of the whitelist contents?

I am not sure what your asking? I read through the script and see that it is part of the whitelist.

When I run the script I am only using -debloat

**Comment by smith581** [comment]
> I have tried to resolve this by re-adding
> Get-AppxPackage -allusers Microsoft.MicrosoftStickyNotes | Foreach { Add-AppxPackage -DisableDevelopmentMode -Register "$($_.InstallLocation)\AppXManifest.xml" }
> at the end of script to reinstall but it doesn't. It shows that it is supposed to be whitelisted but it isnt being saved. Resulting in an sccm redeploy after users called in about it being missed.....

SO with Sticky Notes installed I ran Windows10Debloater.ps1 and it didn't remove Sticky Notes. Great - but this is for upgrade in place cleanup so I don't want prompts. So then I ran Windows10SysPrepDebloater.ps1 Which removed Sticky Notes.
The While list says it isn't to be removed. The ReadMe under "Bloatware that is removed" says it is. Very confusing. I have been using this app  for many years. Most of the time this isnt a deal but the current role requires that Sticky Notes be used. Why isnt the whitelist being applied to Sticky Notes? Windows 10 20H??

**Comment by KenyOS** [comment]
yeah same for me. I'm using default Windows10SysPrepDebloater.ps1 and on Win11, StickyNotes, notepad and paint get deleted. I need to install them on Windows Store to get back these default app.
